src/scss/_variables.scss

```diff
@@ -2,16 +2,15 @@
 $font-color-black: #151515;
 $background-color-white: #f5f5f5;
 $background-color-white-clean: #fff;
-// $grey: #969696;
+$background-color-weight: #f4f4ff;
```

Comment:
Color for what? Rewrite the variable for more understanding, like previously.

Reply:
I need an advice